frontend/src/pages/LineageView.tsx

```diff
@@ -107,7 +121,7 @@ class LineageView extends React.Component<LineageViewProps, LineageViewState> {
             title={`${columnNames[1]}`} />
           <LineageCardColumn
             type='artifact'
-            cards={this.buildArtifactCards([this.state.target])}
+            cards={this.buildArtifactCards([this.state.target], /* isTarget= */ true)}
```

Comment:
Did you mean to remove this after?

Reply:
Added it for readability since the second boolean param name is not obvious. 

http://go/js-style#formatting-param-name-comments

Reply:
Awesome!